packages/extensions/src/node/utils/get-extensions.ts

```diff
@@ -98,58 +99,23 @@ export async function resolvePackageExtensions(root: string, extensionNames?: st
 	return extensions;
 }
 
-export async function getPackageExtensions(root: string): Promise<Extension[]> {
+const isExtension = (pkgName: string) => {
+	const regex = /^(?:(?:@[^/]+\/)?directus-extension-|@directus\/extension-)(.+)$/;
```

Comment:
This is probably not important so feel free to ignore but I think the following might be "more correct" as this disallows multiple leading @ symbols:

```suggestion
	const regex = /^(?:(?:@[^/@]+\/)?directus-extension-|@directus\/extension-)(.+)$/;
```

Reply:
Technically, all non-URL-safe characters are disallowed, but since dependency names are ensured to be valid via package manager / NPM we can be more lax here 👍 

```console
❯ grep name packages/test/package.json 
  "name": "te@st",

❯ pn add -w te@st
 ERR_PNPM_NO_MATCHING_VERSION  No matching version found for te@st
```
(^ although the package "exists" locally)